app/ide-desktop/lib/dashboard/src/components/SvgMask.tsx

```diff
@@ -20,19 +23,22 @@ export interface SvgMaskProps {
   // underlying `div`.
   // eslint-disable-next-line no-restricted-syntax
   readonly className?: string | undefined
-  readonly onClick?: (event: React.MouseEvent) => void
+  readonly onClick?: () => void
 }
 
 /** Use an SVG as a mask. This lets the SVG use the text color (`currentColor`). */
 export default function SvgMask(props: SvgMaskProps) {
-  const { invert = false, alt, src, title, style, color, className, onClick } = props
+  const { invert = false, alt, src, style, color, className, onClick } = props
   const urlSrc = `url(${JSON.stringify(src)})`
   const mask = invert ? `${urlSrc}, linear-gradient(white 0 0)` : urlSrc
+  const ref = React.useRef<HTMLButtonElement>(null)
+
+  const noop = () => {}
 
-  return (
-    <div
-      {...(onClick == null ? {} : { role: 'button' })}
-      title={title}
+  const image = (
+    <aria.Button
```

Comment:
Uh, not sure it's a good idea... Maybe we better leave it as is? SvgMask is more like a presentation component. If we need to add interactivity, we might want to wrap it with a button in place

BTW ariaComponents/Button accepts `icon` prop and looks like we don't need to wrap icon in button at all

Reply:
yeah :skull: it messed up all the E2E tests too